src/applications/mhv-landing-page/tests/components/NavCard.unit.spec.jsx

```diff
@@ -5,72 +5,95 @@ import {
   resolveLandingPageLinks,
   resolveUnreadMessageAriaLabel,
 } from '../../utilities/data';
-import NavCard from '../../components/NavCard';
-
-describe('unread message indicator', () => {
-  function renderCards(unreadMessageCount) {
-    const unreadMessageAriaLabel = resolveUnreadMessageAriaLabel(
-      unreadMessageCount,
-    );
-    const { cards } = resolveLandingPageLinks(
-      undefined,
-      { featureToggles: {} },
-      unreadMessageCount,
-      unreadMessageAriaLabel,
-    );
-
-    return render(
-      cards.map(c => (
-        <NavCard
-          key={c.title}
-          icon={c.icon}
-          title={c.title}
-          links={c.links}
-          aria-label={c.ariaLabel}
-        />
-      )),
-    );
-  }
-
-  const messagesMessage = 'You have unread messages. Go to your inbox.';
-
-  it('includes unread messages message when greater than 0', () => {
-    const unreadMessageCount = 4;
-    const { getByText } = renderCards(unreadMessageCount);
-
-    const inboxSpan = getByText('Go to inbox');
-    const message = inboxSpan.parentNode.getAttribute('aria-label');
-
-    expect(inboxSpan).to.exist;
-    expect(message).to.equal(messagesMessage);
+import NavCard, { externalLinkText } from '../../components/NavCard';
+
+describe('NavCard component', () => {
+  describe('unread message indicator', () => {
```

Comment:
Note: The tests in this section were not changes. This is just an indentation change.